src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -43,6 +45,10 @@ const Row: React.SFC<FlexProps> = ({ children, ...others }) => (
   </Flex>
 )
 
+const ShippingInfoSerif = styled(Serif)`
+  ${unica("s12", "regular")};
```

Comment:
Was there a particular request or need to have this specifically use the `unica` method? In general for the new designs I thought we'd be relying more directly on the palette typography components.